includes/Functions.php

```diff
@@ -0,0 +1,103 @@
+<?php
+/**
+ * Shows a configured message when a payment is successful.
+ * This message can be configured at the Wordpress dashboard.
+ * Also note that the message will be shown
+ * if the short code has been inserted in a page.
+ *
+ * @param $message
+ * @param $track_id
+ * @param $order_id
+ *
+ * @return string
+ * @see \IDPay\CF7\Admin\Menu::admin_table()
+ *
+ */
+function filled_message($message, $track_id, $order_id)
+{
+    return str_replace(["{track_id}", "{order_id}"], [
+        $track_id,
+        $order_id,
+    ], $message);
+}
+
+/**
+ * @param $db
+ * @param $order_id
+ * @param $trans_id
+ * @param $track_id
+ * @param $status
+ * @param $message
+ * @return void
+ */
+function create_callback_response($order_id, $status, $message)
+{
+    global $wpdb;
+    $tableName = 'cf7_callbacks';
+    $row = $wpdb->get_row($wpdb->prepare("SELECT * FROM " . $wpdb->prefix . $tableName . " WHERE id='%s'", $order_id));
+    if ($row == NULL) {
+        $row = [
+            'id' => $order_id,
+            'response' => json_encode(['status' => $status]),
```

Comment:
I said you should change the response to status. it's not necessary to save it as json